keen/client.py

```diff
@@ -492,6 +492,42 @@ def funnel(self, steps, timeframe=None, timezone=None, max_age=None, all_keys=Fa
 
         return self.api.query("funnel", params, all_keys=all_keys)
 
+    def get_collection(self, collection_name):
+        """
+
+        Returns a description of a collection
+
+        :param event_name: the name of the event to get the collection info from
```

Comment:
(Same request as the one on [api.py:118](https://github.com/keenlabs/KeenClient-Python/pull/73/files#diff-31115e53b9da1a9b8ce9f337f6c5285fR218) )